From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 13 Aug 2010 10:35:10 -0400
Subject: [pci] add AMD 10h, 11h PCI IDs to pci_ids.h
Message-id: <20100813103509.4326.16361.stgit@localhost.localdomain>
Patchwork-id: 27554
O-Subject: [RHEL5.6 PATCH 1/3] add AMD 10h,
	11h PCI IDs to include/linux/pci_ids.h
Bugzilla: 443745
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: John Feeney <jfeeney@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=443745

Move PCI device IDs from amd64_edac.h to pci_ids.h where they belong
and add other related ones from upstream.

diff --git a/drivers/edac/amd64_edac.h b/drivers/edac/amd64_edac.h
index b1606c9..3b94ab7 100644
--- a/drivers/edac/amd64_edac.h
+++ b/drivers/edac/amd64_edac.h
@@ -73,13 +73,6 @@
 #include "edac_mc.h"
 #include "edac_mce_amd.h"
 
-#define PCI_DEVICE_ID_AMD_10H_NB_MAP    0x1201
-#define PCI_DEVICE_ID_AMD_10H_NB_DRAM   0x1202
-#define PCI_DEVICE_ID_AMD_10H_NB_MISC   0x1203
-#define PCI_DEVICE_ID_AMD_11H_NB_MAP    0x1301
-#define PCI_DEVICE_ID_AMD_11H_NB_DRAM   0x1302
-#define PCI_DEVICE_ID_AMD_11H_NB_MISC   0x1303
-
 #define amd64_printk(level, fmt, arg...) \
        edac_printk(level, "amd64", fmt, ##arg)
 
diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 16b239b..d9a9e6d 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -490,6 +490,16 @@
 #define PCI_VENDOR_ID_AMD		0x1022
 #define PCI_DEVICE_ID_AMD_K8_NB		0x1100
 #define PCI_DEVICE_ID_AMD_K8_NB_MISC	0x1103
+#define PCI_DEVICE_ID_AMD_10H_NB_HT	0x1200
+#define PCI_DEVICE_ID_AMD_10H_NB_MAP	0x1201
+#define PCI_DEVICE_ID_AMD_10H_NB_DRAM	0x1202
+#define PCI_DEVICE_ID_AMD_10H_NB_MISC	0x1203
+#define PCI_DEVICE_ID_AMD_10H_NB_LINK	0x1204
+#define PCI_DEVICE_ID_AMD_11H_NB_HT	0x1300
+#define PCI_DEVICE_ID_AMD_11H_NB_MAP	0x1301
+#define PCI_DEVICE_ID_AMD_11H_NB_DRAM	0x1302
+#define PCI_DEVICE_ID_AMD_11H_NB_MISC	0x1303
+#define PCI_DEVICE_ID_AMD_11H_NB_LINK	0x1304
 #define PCI_DEVICE_ID_AMD_LANCE		0x2000
 #define PCI_DEVICE_ID_AMD_LANCE_HOME	0x2001
 #define PCI_DEVICE_ID_AMD_SCSI		0x2020
